.tekton/basic-component-pipeline.yaml

```diff
@@ -81,6 +81,10 @@ spec:
     description: Fetch tags with git clone
     name: clone-fetch-tags
     type: string
+  - default: "1d"
+    description: OCI Artifact (workspace) expiration time.
```

Comment:
```suggestion
    description: This sets the expiration time for intermediate OCI artifacts produced and used during builds after which they can be garbage collected.
```

Reply:
done in https://github.com/stackrox/stackrox/pull/11616/commits/4fc31dc77fe56c2f87d4eacb264c93c22a94c9a1